nixos/modules/programs/minipro.nix

```
@@ -8,8 +8,8 @@ in
     programs.minipro = {
       enable = lib.mkEnableOption (lib.mdDoc "minipro") // {
         description = lib.mdDoc ''
-          Installs minipro and its udev rules.
-          Users of the `plugdev` group can interact with connected MiniPRO chip programmers.
+          minipro and its udev rules.
+          Users of the `plugdev` group can interact with connected MiniPRO chip programmers
```

Has the merging thingy again.
Looks like I have missed that in my initial review.

```suggestion
          Whether to enable minipro and its udev rules.
          Users of the `plugdev` group can interact with connected MiniPRO chip programmers.
```

---

Fixed, thanks!